python/lsst/ctrl/bps/pre_transform.py

```diff
@@ -90,8 +93,17 @@ def acquire_quantum_graph(config: BpsConfig, out_prefix: str = "") -> tuple[str,
 
     _LOG.info("Reading quantum graph from '%s'", qgraph_filename)
     with time_this(log=_LOG, level=logging.INFO, prefix=None, msg="Completed reading quantum graph"):
-        qgraph = QuantumGraph.loadUri(qgraph_filename)
-
+        qgraph_filename = ResourcePath(qgraph_filename)
+        if qgraph_filename.getExtension() == ".qg":
+            with PredictedQuantumGraph.open(
+                qgraph_filename, import_mode=TaskImportMode.DO_NOT_IMPORT
+            ) as reader:
+                reader.read_thin_graph()
+                qgraph = reader.finish()
+        elif qgraph_filename.getExtension() == ".qgraph":
+            qgraph = PredictedQuantumGraph.from_old_quantum_graph(QuantumGraph.loadUri(qgraph_filename))
+        else:
+            raise ValueError(f"Unrecognized extension for quantum graph file: {qgraph_filename}.")
     return qgraph_filename, qgraph
```

Comment:
Just a note that leaving the file a .qgraph file will mean every compute job needs to convert it.  If this turns out to be more common than I expect, we can make another ticket to have the new format here and save the new filename so that the rest of the submission and running uses the new format file.

Reply:
Also ctrl_bps expects the returned qgraph_filename to be a str, so needs to be changed to `qgraph_filename.ospath`